tools/Dockerfile

```diff
@@ -73,7 +73,7 @@ RUN set -eux; \
     rustup --version; \
     rustup component add rustfmt; \
     rustup component add clippy; \
-    rustup install ${rust_nightly_version}; \
+    rustup toolchain install ${rust_nightly_version} --component clippy; \
```

Comment:
side note: we might want to be explicit about the toolchains we install. I've run into some surprises from time to time relying on the defaults (and this might also install components we _don't_ need)

Reply:
https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file may also be handy

Reply:
I think the `--profile minimal` on `rustup-init` is making sure it won't install any undesired components.